tests/test_transferlearning.py

```diff
@@ -0,0 +1,246 @@
+import numpy as np
+from numpy.testing import assert_array_equal
+from pyriemann.transferlearning import MDWM
+import pytest
+
+
+mat_transfer = [MDWM]
+
+
+@pytest.mark.parametrize("transfer", mat_transfer)
+class TransferTestCase:
```

Comment:
can you avoid using classes for tests and use plain functions and fixtures to share code between tests?

Reply:
I agree that in this case it could be plain function as there is only one model to test. The goal is to have a TestCase class to make the tests for the common sklearn API functions (fit, predict, predict_proba, ...), as we have done in [test_classification](https://github.com/pyRiemann/pyRiemann/blob/master/tests/test_classification.py#L26) and other test file. The tests for specific aspects of the model are in plain function and use fixtures.

I'll update the code to make it clearer.